tests/libtest/test_ibmcloud.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf8 -*-
+
+
+import logging
+
+from ocs_ci.ocs.platform_nodes import IBMCloud
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import libtest
+from ocs_ci.ocs import constants, node
+from ocs_ci.ocs.node import get_all_nodes
+
+from ocs_ci.utility.utils import run_cmd
+
+logger = logging.getLogger(__name__)
+
+
+@libtest
```

Comment:
These lib tests are compatible only for IBM Cloud platform so they should be skipped if the platform is not IBM Cloud. The same for other lib tests as well.

Reply:
written the ibmcloud_platform_required and runs the testcases only on ibmcloud platform